articles/100_ament.md

```diff
@@ -20,34 +20,93 @@ author: '[Dirk Thomas](https://github.com/dirk-thomas)'
+
+
 ## ament
 
-`ament` is a meta build system to build applications which are split into separate packages.
+ament is a meta build system to improve building applications which are split into separate packages.
+It consists of two major parts:
 
-It relies on meta information about the packages to determine their build dependencies.
-This meta information is specified in a file called `package.xml` which is defined in [REP 140](http://www.ros.org/reps/rep-0140.html).
+* the *build system* to configure, build, and install a single package
+* a *tool* to invoke the build of individual packages in their topological order
 
-ament consists of several separate parts:
+The tool relies on meta information about the packages to determine their dependencies.
+This meta information is defined in a manifest file called `package.xml` which is specified in [REP 140](http://www.ros.org/reps/rep-0140.html).
 
-* [ament_package](https://github.com/ament/ament_package) is a Python package providing an API to parse the files containing the meta information.
-* [ament_cmake](https://github.com/ament/ament_cmake) contains a set of packages which provide CMake based functionality.
-  The features are split across several packages in order to ensure that they are cleanly separated from each other.
-  The main feature of the CMake packages is to automatically generate a CMake config file to pass information to downstream packages.
-* [ament_lint](https://github.com/ament/ament_lint) contains a set of packages which perform linting tasks.
-  For a set of linters a ROS 2 specific configuration is provided which can be used via the command line as well as from within CMake (commonly as a CTest).
-* [ament_tools](https://github.com/ament/ament_tools) is a Python package which provides command line tools to build, test and install packages.
-  It is similar to [catkin_tools](https://github.com/catkin/catkin_tools) and builds each package in a workspace in topological order.
-  Each different build system is integrated through extension points.
-  While it currently does not build packages in parallel that will be added in the future to speed up the build process.
-* [ament_index](https://github.com/ament/ament_index) contains a set of packages which provide API to access the [ament resource index](https://github.com/ament/ament_cmake/blob/master/ament_cmake_core/doc/resource_index.md).
-  Currently it only contains a Python implementation but other languages like C++ should follow.
+Each package is build separately with its own build system.
+In order to make the output of one package available to other packages each package can extend the environment in a way that downstream packages can find and use its artifacts and resources.
+If the resulting artifacts are installed into e.g. `/usr` it might not be necessary to alter the environment at all since these folders are commonly being searched by various tools.
```

Comment:
Using `e.g.` inline like that doesn't read well in my opinion: ```... are installed into `/usr`, for example, it might not ...``` or start with it: ```For example, if the resulting artifacts are installed into `/usr` it might ...```